frescobaldi_app/main.py

```diff
@@ -83,6 +85,7 @@ def parse_commandline():
         parser.add_argument('-p', '--port')
         parser.add_argument('-f', '--file')
         parser.add_argument('-o', '--output')
+        parser.add_argument('-y', '--python-ly')
```

Comment:
Will there not be a conflict for the `-p` argument when pydevd is used, and do you need to add the python ly argument again?

Reply:
I'm sorry but I don't really understand:
* I have named the short argument `-y` in order to avoid that conflict
* Why "again"? I don't see where it has already been used

Reply:
OK, only now I see what you meant. I have clarified in cc40d5b5

Reply:
Maybe it's me that don't understand! :slightly_smiling_face: As I read it you still add it twice! 

From the code and from your comments I assume you see it as two different sets of arguments and that you need to add it to both. Why is that? :thinking: 

I have searched for a relevant reference about this in the pydevd docs but haven't found any!